editor/editor_atlas_packer.cpp

```diff
@@ -59,7 +59,7 @@ void EditorAtlasPacker::chart_pack(Vector<Chart> &charts, int &r_width, int &r_h
 
 		Ref<BitMap> src_bitmap;
 		src_bitmap.instantiate();
-		src_bitmap->create(aabb.size / divide_by);
+		src_bitmap->create((aabb.size / divide_by).max(Vector2i(1, 1)));
```

Comment:
I get this gets rid of the zero-size errors. But does the algorithm work as it's supposed to in such cases? Or was made just to silence the errors?

Reply:
It does as far as I can tell, this is for the cells for checking overlap as far as I can tell, and polygons with a AABB smaller than 4 on a side (the standard cell size) should still be valid input I think